src/components/Header/Header.js

```diff
@@ -1,9 +1,106 @@
-import React from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import './Header.scss';
+import HeaderNavigation from './HeaderNavigation/HeaderNavigation';
+import { useDispatch } from 'react-redux';
+import { initSubMenuStatues } from './HeaderNavigation/HeaderNavigationSlice';
+import HeaderLogo from './HeaderLogo/HeaderLogo';
+import HeaderLinks from './HeaderLinks/HeaderLinks';
+import HeaderContacts from './HeaderContacts/HeaderContacts';
+import SearchButton from './SearchButton/SearchButton';
 
 const Header = (props) => {
-  return <header className={`header ${props.className ? props.className : ''}`}>Header</header>;
+  const [navItems] = useState([
+    {
+      link: '#',
+      title: 'Sales',
+
+      submenu: [
+        {
+          link: '/sellers/link/',
+          title: 'Western Suburbs',
+        },
+        { link: '/sellers/link/', title: 'North Shore' },
+        { link: '/sellers/link/', title: 'South Shore' },
+        {
+          link: '/sellers/link/',
+          title: 'Cape Cod & Islands',
+        },
+        {
+          link: '/sellers/link/',
+          title: 'Off Market Listings',
+        },
+      ],
+    },
+    {
+      link: '#',
+      title: 'Rentals',
+
+      submenu: [
+        {
+          link: '/sellers/link/',
+          title: 'Western Suburbs',
+        },
+        { link: '/sellers/link/', title: 'North Shore' },
+        { link: '/sellers/link/', title: 'South Shore' },
+        {
+          link: '/sellers/link/',
+          title: 'Cape Cod & Islands',
+        },
+        {
+          link: '/sellers/link/',
+          title: 'Off Market Listings',
+        },
+      ],
+    },
+    { link: '/', title: 'Quick search' },
+    { link: '/', title: 'Neighborhoods' },
+    {
+      link: '/',
+      title: 'About',
+
+      submenu: [
+        { link: '/sellers/link/', title: 'Team' },
+        { link: '/sellers/link/', title: 'Contact' },
+      ],
+    },
+  ]);
+  const [logoInfo] = useState({
+    link: '/',
+    src: 'https://s3.amazonaws.com/files.usmre.com/8844/usm-logo-blue.svg',
+    alt: 'Logo',
+  });
+  const [headerLinks] = useState([
+    { title: 'My Favorites', link: '/listings/favorite/' },
+    { title: 'Contact', link: '/contact-us/' },
+    { title: 'Log In', link: '#' },
+    { title: 'Sign Up', link: '#' },
+  ]);
+  const [contacts] = useState([{ title: '132-132-1324', link: 'tel:1321321324' }]);
+  const dispatch = useDispatch();
+  const navItemsNumber = navItems.length ? navItems.length : 0;
+  const subMenuStatuses = new Array(navItemsNumber).fill(false);
+
+  useEffect(() => {
+    dispatch(initSubMenuStatues(subMenuStatuses));
+  }, []);
+
+  return (
+    <header className={`header ${props.className ? props.className : ''}`}>
+      <div className="container">
+        <div className="header__part header__part_left">
```

Comment:
Because of the way you are using this, I think this should be className="header__part header__part--left". Where 'left' is a modifier to the header part. Same for header__part_right below.